src/easier68k/simulator/m68k.py

```diff
@@ -67,3 +67,18 @@ def clear_cycles(self):
         :return:
         """
         self._clock_cycles = 0
+
+    def load_memory(self, file):
```

Comment:
can we add type specification for `file`?

Reply:
Yup